cds/modules/redirector/views.py

```diff
@@ -49,17 +47,16 @@
 
 
 def recid_from_rn(report_number):
-    """Retrieve a report number's corresponding record ID."""
+    """Retrieves a report number's corresponding record ID."""
     object_uuid = PersistentIdentifier.query.filter_by(
         pid_type='rn',
         pid_value=report_number
     ).one().object_uuid
```

Comment:
We have a problem here if the PID has been redirected, maybe we should use a resolver instead of accessing directly to the models.